src/dev/license_checker/config.ts

```diff
@@ -69,6 +69,8 @@ export const LICENSE_ALLOWED = [
   'Python-2.0',
   '(Apache-2.0 AND MIT)',
   'BlueOak-1.0.0',
+  '(MPL-2.0 OR Apache-2.0)',
+  'EPL-2.0',
```

Comment:
Mermaid is [MIT](https://github.com/mermaid-js/mermaid/blob/develop/LICENSE), so why the license update here? Separate dependency?

Reply:
correct, one is coming from `dompurify` and other is for `elkjs`
https://buildkite.com/elastic/kibana-pull-request/builds/164277#018aece4-0014-43b4-9ad8-7d3647b04b59
![Uploading image.png…]()